doc/api/fs.md

```diff
@@ -1225,6 +1225,9 @@ changes:
 Change the file timestamps of a file referenced by the supplied file
 descriptor.
 
+*Note*: This function does not work on AIX versions older than 7.1. It can still
+be called but will return `UV_ENOSYS`.
+
```

Comment:
How about:

*Note*: This function does not work on AIX versions before 7.1, 
it will return the error `UV_ENOSYS`.


While I agree older probably means the right thing the numbering is opposite to what usually comes to mind with older (ie 72 is older than 71  in years)